plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/PipelineStageViewHook.java

```diff
@@ -23,7 +23,7 @@ protected String getParentProjectName() {
 
     @Override
     protected String getPluginFolderName(UpdateSite.Plugin currentPlugin){
-        return "rest-api";
+        return (currentPlugin.getDisplayName() == "pipeline-rest-api") ? "rest-api" : "ui";
```

Comment:
Are you sure you want `.getDisplayName` here instead of `.name`? I think the display name is something like "Pipeline Stage View", whereas `.name` is the artifactId, but I'm not sure.

Reply:
Oh, also you shouldn't compare `String`s using `==`, you should use `.equals()`. I think the check is always false right now.

Reply:
Changed!! I was reviewing and the proper way to get the name is using the getDisplayName(), it is returned properly like `pipeline-rest-api`